test/OneDriveAuth.test.js

```diff
@@ -302,6 +376,7 @@ describe('OneDriveAuth Tests', () => {
           beforeCacheAccess: () => {
             baseRefreshed = true;
           },
+          getPluginMetadata: () => {},
```

Comment:
the mock base used here needs that method as well (or we need to guard the call to `base.getPluginMetadata` in `MemCachePlugin`)